### ISSUE TITLE
adjust transferred perfomance

### ISSUE THREAD

**Issue by stihuangyuan** [GaParmar/img2img-turbo#156]
Hello, I'm using img2img-turbo unpaired data to train a model for daylight-to-rainy style generation. 
I'm currently experiencing a problem with my testing: 
when I add a test image containing sunlight and shadows, the generated image also generates sunlight and shadows. 
How can I adjust the model to prevent shadows from being transferred to the generated image? Should I modify the training data or the training loss?
